specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/preview/2021-09-01-preview/galleryImages.json

```diff
@@ -0,0 +1,590 @@
+        }
+      },
+      "required": [
+        "storageProfile"
+      ],
+      "description": "Describes the properties of a gallery image version."
+    },
+    "galleryImageVersionStorageProfile": {
+      "type": "object",
+      "properties": {
+        "osDiskImage": {
+          "$ref": "#/definitions/galleryOSDiskImage"
+        }
+      },
+      "description": "This is the storage profile of a Gallery Image Version."
+    },
+    "galleryOSDiskImage": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/galleryDiskImage"
+        }
+      ],
+      "description": "This is the OS disk image."
+    },
+    "galleryDiskImage": {
+      "type": "object",
+      "properties": {
+        "sizeInMB": {
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true,
+          "description": "This property indicates the size of the VHD to be created."
+        }
+      },
+      "description": "This is the disk image base class."
+    },
+    "galleryimages": {
+      "type": "object",
+      "description": "The galleryimages resource definition.",
+      "x-ms-azure-resource": true,
+      "x-ms-client-flatten": true,
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/galleryimagesProperties"
+        },
+        "extendedLocation": {
+          "type": "object",
+          "description": "The extendedLocation of the resource.",
+          "properties": {
+            "type": {
+              "type": "string",
+              "description": "The extended location type."
+            },
+            "name": {
+              "type": "string",
+              "description": "The extended location name."
+            }
+          }
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/TrackedResource"
+        }
+      ]
+    },
+    "galleryimagesPatch": {
+      "type": "object",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        }
+      },
+      "description": "The galleryimages resource patch definition."
+    },
+    "galleryimagesListResult": {
+      "type": "object",
+      "description": "List of galleryimages.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/galleryimages"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Link to the next set of results.",
+          "readOnly": true
+        }
+      }
+    }
+  },
+  "parameters": {
+    "galleryimagesNameParameter": {
```

Comment:
>galleryimagesNameParameter [](http://example.com/codeflow?start=5&length=26)

suggest calling it GalleryImageNameParameter

Reply:
Fixed